ios/Example/Tests/CertificateTests.swift

```diff
@@ -0,0 +1,61 @@
+//
+//  CertificateTests.swift
+//  Trifle_Tests
+//
+//  Created by Gelareh Taban on 5/17/23.
+//  Copyright © 2023 CocoaPods. All rights reserved.
+//
+
+import Foundation
+import XCTest
+import Wire
+@testable import Trifle
+
+final class CertificateTests: XCTestCase {
```

Comment:
I think these unit tests are functionally the same as TrifleCertificateTests. Should we need this?

Reply:
Since we have abstracted away the classes,  I think separate tests are good to have.